src/patterns/error-validation/examples/errors-proto/errors/index.njk

```diff
@@ -170,6 +170,7 @@ layout: none
                         "value": "other",
                         "other": {
                             "open": true,
+                            "required": true,
```

Comment:
The errors proto doesn't current do any kind of form post, but it would be good if we could do that.  So this would work.